modules/administration-guide/examples/snip_che-installing-che-in-a-restricted-environment-step-disable-signature-checks.adoc

```diff
@@ -0,0 +1,16 @@
+// included in:
+// modules/administration-guide/partials/snip_preparing-images-for-a-restricted-environment.adoc
+
+. To avoid checking signatures for the `community-operator-index` catalog image, edit the `/etc/containers/policy.json` file:
```

Comment:
We should specify that it should be done "on your workstation" as for the other commands.